Pully Penelope/Assets/Scripts/Guillotine.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Guillotine : MonoBehaviour
+{
+    private GameObject playerCharacter;
+    
+    [SerializeField]
+    private float offset;
+    private SpriteRenderer spriteRenderer;
+
+    private void Start()
+    {
+        playerCharacter = GameObject.FindWithTag("Player");
```

Comment:
if your player exists in the scene before runtime, you can probably just make it a serialized field and drag it in in the inspector instead of having to search for it using FindWithTag